src/Forms/GridField/GridFieldDetailForm_ItemRequest.php

```diff
@@ -930,11 +931,13 @@ public function Breadcrumbs($unlinked = false)
         $items = $this->popupController->Breadcrumbs($unlinked);
 
         if (!$items) {
+            /** @var ArrayList<ArrayData> $items */
             $items = ArrayList::create();
         }
 
-        if ($this->record && $this->record->ID) {
-            $title = ($this->record->Title) ? $this->record->Title : "#{$this->record->ID}";
+        $record = $this->getRecord();
+        if ($record && $record->ID) {
+            $title = ($record->Title) ? $record->Title : "#{$record->ID}";
```

Comment:
Refactoring slightly so I can use the `$record` variable lower down - also updating this to use best practice (public getter better than protected property)